### DISCUSSION TITLE
Why can't we easily disable Copilot?

### DISCUSSION THREAD

**Discussion by Kyliathy** [community/community#113531] [Copilot Conversations]
### Select Topic Area

Question

### Body

Copilot can get very annoying when I type out comments. Currently, one can disable it only via several mouse movements/clicks. When can we expect a way to easily disable it with a hotkey? 

I am using VS Code, but the question does, in fact, apply to all IDEs, because I think this should be a feature any developer using any IDE should have.

**Comment by SebastienXia** [comment]
## Hello, @Kyliathy 

You can do a keyboard shortcut with Copilot in VScode.

![Capture](https://github.com/community/community/assets/105004912/f70ffb77-6241-4b7e-828f-248c464ad891)

**Comment by Kyliathy** [comment]
Hi :). Unfortunately, that doesn't work.  Tried it already. Seems to be bugged out.

**Comment by ppcamp** [comment]
You may try add `"github.copilot.editor.enableAutoCompletions": false,` to your vscode config. You can also disable copilot basing on languages

```
 "github.copilot.enable": {
        "markdown": "false"
    }
```

**Comment by Kyliathy** [comment]
Thank you for the suggestion :) but it has no connection with the actual "request" (at this point might even call it a bug report). 
That's because it defeats the purpose :). I want to do it via hotkey. And hotkeys for different languages aren't even available.
This switch has to be FAST to use, because at one moment I'm commenting and don't need it, and in 5 seconds I'm back to coding and I do need it.

**Comment by Kyliathy** [comment]
It's fixed in the latest version! :)

**Reply by Captain-Leftovers** [reply]
i cant get it to work can you elaborate what is fixed because when i toggle enable/disable completions nothing happens i still get completions lol

**Comment by lzandman** [comment]
The way I see it, there should be a dedicated button for it in the UI (which of course could also be mapped to a key binding). There are times that you just don't want Copilot completions and/or any risk on stuff getting sent to the Copilot cloud.
